internal/types/document.go

```diff
@@ -229,6 +294,14 @@ func (d *Document) Get(key string) (any, error) {
 	return nil, fmt.Errorf("types.Document.Get: key not found: %q", key)
 }
 
+func (d *Document) MustGet(key string) any {
```

Comment:
It appears to be unused.

And my idea was to use `must.NotFail(...)` instead of adding more MustXXX functions methods.

Reply:
removed